raiden/raiden_service.py

```diff
@@ -363,29 +363,32 @@ def start(self):
         self.alarm.register_callback(self._callback_new_block)
         self.alarm.first_run()
 
+        chain_state = views.state_from_raiden(self)
+
+        self._initialize_transactions_queues(chain_state)
+        self._initialize_whitelists(chain_state)
+
+        # send messages in queue before starting transport,
+        # this is necessary to avoid a race where, if the transport is started
+        # before the messages are queued, actions triggered by it can cause new
+        # messages to be enqueued before these older ones
+        self._initialize_messages_queues(chain_state)
+
         # The transport must not ever be started before the alarm task's first
```

Comment:
How does this comment reconcile with the fact that the transport is started right above the alarm task?